src/fe/fe_interface_inf_fe.C

```diff
@@ -425,6 +425,30 @@ void FEInterface::ifem_nodal_soln(const unsigned int dim,
 
 
 
+Point FEInterface::ifem_map (const unsigned int dim, 
+                             const FEType& fe_t,
+                             const Elem * elem,
+                             const Point & p){
+   switch (fe_t.inf_map){
```

Comment:
Curly braces in this entire function need to go on their own line to match our coding conventions.